### ISSUE TITLE
Lists with nested block elements get messed up regulary (inline editor)

### ISSUE THREAD

**Issue by TinyMCE-User** [tinymce/tinymce#1731]
When you have a list with nested block elements (easily happens with paste) additional newlines appear when you remove/outdent the list.
1. make a list
2. format (parts of) the text as paragraph
3. select the list
4. remove the list (or outdent)

In the source (Firebug or something) there are empty paragraphs (&ltg;p>&ltg;/p>) now.
With view source code in tinyMCE or save there are &nbsp; added inside these elements and they appear as empty lines.

In addition the first list element (with nested block format) remains within the list if you select several list elements and outdent / remove the list.
Especially in IE you can also get staggered lists (with pasting inside lists). Most times you will see this only after a save and reload or a view source.

Personally I never needed nor wanted block formats inside a list in any project.

> _**Legacy information imported from TinyMCE bug tracker:**_
> 
> _[#T6991](http://archive.tinymce.com/develop/bugtracker_view.php?id=6991) posted by [pernille](http://archive.tinymce.com/profile.php?name=pernille)_
> 
> _**Tags:** []_
> _**Status:** Open_
> _**Resolution:** None_
> _**Attached URL:** none_

**Comment by fyrkant** [comment]
Closing this due to old age, if anyone still has this issue in the latest version of TinyMCE please open up a new issue.
